examples/fizzbuzz/package.yaml

```diff
@@ -0,0 +1,39 @@
+name:                fizzbuzz
+version:             0.1.0.0
+github:              "githubuser/fizzbuzz"
+license:             BSD3
+author:              "Jorge Guerra"
+maintainer:          "cmotoche@stackbuilders.com"
+copyright:           "2019 StackBuilders"
+
+extra-source-files:
+- README.md
+
+# Metadata used when publishing your package
+# synopsis:            Short description of your package
+# category:            Web
+
+# To avoid duplicated efforts in documentation and dealing with the
+# complications of embedding Haddock markup inside cabal files, it is
+# common to point users to the README.md file.
```

Comment:
@Jagl257 Let's remove this comments. I don't think they're necessary.

Reply:
Ok @CristhianMotoche, done.